.github/workflows/build_reusable.yml

```diff
@@ -133,6 +133,11 @@ jobs:
       # clean up any previous artifacts to avoid hitting disk space limits
       - run: git clean -xdf && rm -rf /tmp/next-repo-*; rm -rf /tmp/next-install-* /tmp/yarn-* /tmp/ncc-cache target
 
+      - name: Set CI git user for CNA
```

Comment:
CVN numbering authority? Why is this needed now?

Reply:
"Create Next App", maybe I should spell it out?

This is needed for the added integration test expectation so check whether there are any git changes after creating a new project and running `next dev`. Without this, the `git init` silently fails because no git user is defined.

Reply:
Always good practice to spell out abbreviations on their first usage.

Reply:
Agreed, fixed it by adding spelling it out, and adding a comment above.